domain/src/main/java/com/eleks/domain/intearactor/MotivationPhraseInteractor.kt

```diff
@@ -1,6 +1,37 @@
 package com.eleks.domain.intearactor
 
+import com.eleks.domain.model.GroupPhraseModel
 import com.eleks.domain.repository.MotivationPhraseRepository
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
 
 class MotivationPhraseInteractor(private val motivationPhraseRepository: MotivationPhraseRepository) {
+
+    fun getGroupPhraseListFlow(): Flow<List<GroupPhraseModel>> {
+        return flow {
+            delay(1500)
```

Comment:
Do we still need delay?

Reply:
This is just temporary delay until we add final BL